src/client/src/tscUtil.c

```diff
@@ -3068,6 +3068,12 @@ int32_t tscValidateName(SStrToken* pToken, bool escapeEnabled, bool *dbIncluded)
       }
     }
 
+    if (escapeEnabled && pToken->type == TK_ID) {
```

Comment:
The first part of a full table name, i.e. dbname, may be enclosed with backward-quotation marks "`".